pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTVariableId.java

```diff
@@ -135,7 +135,7 @@ private ModifierOwner getModifierOwnerParent() {
      * identifier.
      */
     public boolean isUnnamed() {
-        return "_".equals(name) && getLanguageVersion().compareToVersion("21-preview") >= 0;
+        return "_".equals(name) && getLanguageVersion().compareToVersion("21") >= 0;
```

Comment:
```suggestion
        return "_".equals(name) && getLanguageVersion().compareToVersion("22") >= 0;
```

It is a 21-preview or 22 release feature.